tests/components/upnp/test_init.py

```diff
@@ -122,9 +133,9 @@ def udn(self):
         assert hass.data[upnp.DOMAIN]["devices"][udn] == mock_device
 
         # ensure add-port-mapping-methods called
-        assert mock_device.added_port_mappings == [
-            [8123, ip_address("192.168.1.10"), 8123]
-        ]
+        assert mock_device.added_port_mappings == [[
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/app/foreign-sub/home-assistant/pullRequest?prid=4643140)